packages/cookie-banner/src/lib/ui.js

```diff
@@ -125,6 +125,10 @@ export const initForm = (Store, track = true) => state => {
             ]
         );
     });
+
+    if(window.location.hash.substring(1) === form.id) {
+        window.scrollTo(0, form.offsetTop);
```

Comment:
(I had to look it up to confirm but) offsetTop returns the number of pixels from the closest relatively positioned parent. https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetTop#parameters

Maybe form.getBoundingClientRect().top + window.scrollY would be more robust?

Reply:
Ah yes of course - the example had next to no CSS so it seemed to be working at the time.  Updated, thanks!